src/omg-cli/omg/commands/cleanup_project.go

```diff
@@ -93,9 +93,10 @@ func runAsync(steps []step) error {
 	for _, step := range steps {
 		step := step
```

Comment:
not your code, but this is strange

Reply:
I can't see why this copy was needed. I removed it in my latest commit.